templates/resources/fabric_connection.md.tmpl

```diff
@@ -94,6 +94,49 @@ EVPLAN Port to Network Connection:
 
 {{tffile "examples/resources/equinix_fabric_connection/fcr_to_metal.tf"}}
 
+Port to Alibaba Connection EVPL_VC Connection:
+
+{{tffile "examples/resources/equinix_fabric_connection/port_to_alibaba.tf"}}
+
+### Step by Step Instructions for Fabric Port to Fabric Alibaba Profile Connection
+
+#### 1. Create Connection from Equinix Terraform Provider
+
+**Note:** The `connection_name` must follow either one of these patterns:
+
+* A unique `connection_name` with atmost 24 characters
+`connection_name = var.connection_name`
+* A `connection_name` with atmost 12 characters combined with a random string of 12 characters:
+  `connection_name = "${var.connection_name}${random_string.random.result}"`
+  Use the `random_string` resource to perform this operation:
+  ```hcl
+  resource "random_string" "random" {
+    length  = 12
+    special = false
+  }
+  ```
+  * A unique `connection_name` is generated by appending a 12-character random string to the variable name given by the user
+  * A Fabric Port to Alibaba Profile Connection is successfully created and displayed in the Fabric Portal
+  * An Express Connect Physical Connection resource is created and becomes visible in the Alibaba Portal
+
+#### 2. Formulate main.tf and output.tf Files
```

Comment:
@thogarty - When we discussed, were you referring to changing the words in this step? (Currently, it is the same as in `README.md` of `terraform-equinix-fabric`)